external-crates/move/crates/move-analyzer/trace-debug/package.json

```diff
@@ -5,7 +5,7 @@
     "publisher": "mysten",
     "icon": "images/move.png",
     "license": "Apache-2.0",
-    "version": "0.0.8",
+    "version": "0.0.9",
```

Comment:
Will publish debug extension version that contains PTB-related stuff at the same time as this PR lands